Modix.Services/Notes/NoteRemoverService.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Modix.Services.Notes
+{
+    using System;
+    using System.Threading.Tasks;
+    using Data;
+    using Data.Models.Moderation;
+    using Serilog;
+
+    public class NoteRemoverService : INoteRemoverService
+    {
+        private readonly ModixContext _modixContext;
+
+        public NoteRemoverService(ModixContext modixContext)
+        {
+            _modixContext = modixContext;
+        }
+
+        public async Task<bool> Remove(NoteEntity note)
+        {
+            try
+            {
+                _modixContext.Notes.Remove(note);
+                var rowsChanged = await _modixContext.SaveChangesAsync();
```

Comment:
Will this work if the `NoteEntity` isn't attached to the `DbContext`? I know EF is weird about that for something operations.

Reply:
Really `Entity` objects shouldn't be exposed directly to higher layers, at least that's the way I've setup all the other features. You only interact the the database through a `Repository` that controls what operations you can and can't perform pretty tightly, since EF queries can easily bleed across services, and cause unintended performance issues.